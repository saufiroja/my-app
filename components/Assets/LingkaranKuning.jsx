import React from "react";
import Image from 'next/image';

const LingkaranKuning = () => {
  return (
    <div className="absolute xl:left-[50%] xl:top-[20%] xl:flex lg:left-[53%] lg:top-[17%] lg:flex md:hidden sm:hidden">
        <Image src="/images/kuning.svg" alt="Lingkaran-Kuning" width={120} height={120}/>
    </div>
  );
};

export default LingkaranKuning;
