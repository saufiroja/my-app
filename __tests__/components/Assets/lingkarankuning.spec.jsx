import React from 'react';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import LingkaranKuning from '../../../components/Assets/LingkaranKuning';

describe('LingkaranKuning', () => {
  it('renders a image', () => {
    render(<LingkaranKuning />);

    const image = screen.getByRole('img', {
      name: 'Lingkaran-Kuning',
    });

    const incrediblesPosterImg = screen.getByAltText('Lingkaran-Kuning');

    expect(image).toBeInTheDocument();
    expect(incrediblesPosterImg).toBeInTheDocument();
  });

  it('matches the snapshot', () => {
    const tree = renderer.create(<LingkaranKuning />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
