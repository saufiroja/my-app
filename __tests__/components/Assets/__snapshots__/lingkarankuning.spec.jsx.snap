// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LingkaranKuning matches the snapshot 1`] = `
<div
  className="absolute xl:left-[50%] xl:top-[20%] xl:flex lg:left-[53%] lg:top-[17%] lg:flex md:hidden sm:hidden"
>
  <span
    style={
      Object {
        "background": "none",
        "border": 0,
        "boxSizing": "border-box",
        "display": "inline-block",
        "height": "initial",
        "margin": 0,
        "maxWidth": "100%",
        "opacity": 1,
        "overflow": "hidden",
        "padding": 0,
        "position": "relative",
        "width": "initial",
      }
    }
  >
    <span
      style={
        Object {
          "background": "none",
          "border": 0,
          "boxSizing": "border-box",
          "display": "block",
          "height": "initial",
          "margin": 0,
          "maxWidth": "100%",
          "opacity": 1,
          "padding": 0,
          "width": "initial",
        }
      }
    >
      <img
        alt=""
        aria-hidden={true}
        src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27120%27%20height=%27120%27/%3e"
        style={
          Object {
            "background": "none",
            "border": 0,
            "display": "block",
            "height": "initial",
            "margin": 0,
            "maxWidth": "100%",
            "opacity": 1,
            "padding": 0,
            "width": "initial",
          }
        }
      />
    </span>
    <img
      alt="Lingkaran-Kuning"
      data-nimg="intrinsic"
      decoding="async"
      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
      style={
        Object {
          "border": "none",
          "bottom": 0,
          "boxSizing": "border-box",
          "display": "block",
          "height": 0,
          "left": 0,
          "margin": "auto",
          "maxHeight": "100%",
          "maxWidth": "100%",
          "minHeight": "100%",
          "minWidth": "100%",
          "objectFit": undefined,
          "objectPosition": undefined,
          "padding": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "width": 0,
        }
      }
    />
    <noscript>
      <img
        alt="Lingkaran-Kuning"
        data-nimg="intrinsic"
        decoding="async"
        loading="lazy"
        src="/images/kuning.svg"
        srcSet="/images/kuning.svg 1x, /images/kuning.svg 2x"
        style={
          Object {
            "border": "none",
            "bottom": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": 0,
            "left": 0,
            "margin": "auto",
            "maxHeight": "100%",
            "maxWidth": "100%",
            "minHeight": "100%",
            "minWidth": "100%",
            "objectFit": undefined,
            "objectPosition": undefined,
            "padding": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "width": 0,
          }
        }
      />
    </noscript>
  </span>
</div>
`;
